openshift/template.yaml

```diff
@@ -7,329 +7,330 @@ metadata:
-                  valueFrom:
-                    secretKeyRef:
-                      name: golang-insights-s3
-                      key: aws_access_key_id
-                - name: POSTGRESQL_DATABASE
-                  valueFrom:
-                    secretKeyRef:
-                      key: database
-                      name: coreapi-postgres
-                - name: POSTGRESQL_PASSWORD
-                  valueFrom:
-                    secretKeyRef:
-                      key: password
-                      name: coreapi-postgres
-                - name: POSTGRESQL_USER
-                  valueFrom:
-                    secretKeyRef:
-                      key: username
-                      name: coreapi-postgres
-                - name: SENTRY_AUTH_TOKEN
-                  valueFrom:
-                    secretKeyRef:
-                      name: worker
-                      key: sentry-auth-token
-                - name: SENTRY_URL
-                  valueFrom:
-                    secretKeyRef:
-                      name: worker
-                      key: sentry-url
-                - name: REPORT_BUCKET_NAME
-                  valueFrom:
-                    secretKeyRef:
-                      name: developer-analytics-audit-report-s3
-                      key: bucket
-                - name: NPM_MODEL_BUCKET
-                  valueFrom:
-                    secretKeyRef:
-                      name: cvae-npm-insights-s3
-                      key: bucket
-                - name: MAVEN_MODEL_BUCKET
-                  valueFrom:
-                    secretKeyRef:
-                      name: hpf-maven-insights-s3
-                      key: bucket
-                - name: PYPI_MODEL_BUCKET
-                  valueFrom:
-                    secretKeyRef:
-                      name: hpf-pypi-insights-s3
-                      key: bucket
-                - name: GOLANG_MODEL_BUCKET
-                  valueFrom:
-                    secretKeyRef:
-                      name: golang-insights-s3
-                      key: bucket
-                - name: ENCRYPTION_KEY_FOR_SNYK_TOKEN
-                  valueFrom:
-                    secretKeyRef:
-                      name: snyk-secrets
-                      key: encryption_key_for_snyk_token
-                - name: MANIFESTS_BUCKET
-                  valueFrom:
-                    secretKeyRef:
-                      name: dynamic-manifests
-                      key: bucket
-                - name: APP_SECRET_KEY
-                  valueFrom:
-                    secretKeyRef:
-                      name: jobs
-                      key: flask-app-secret-key
-              resources:
-                requests:
-                  memory: ${MEMORY_REQUEST}
-                  cpu: ${CPU_REQUEST}
-                limits:
-                  memory: ${MEMORY_LIMIT}
-                  cpu: ${CPU_LIMIT}
+  - apiVersion: batch/v1beta1
```

Comment:
What is the change in this file, it seems like the whole file has changed.

Reply:
Only the env var for recommendation_feedback has been addded and the retaining value for SA table has been chnaged to 181